kebechet/cli.py

```diff
@@ -91,5 +90,13 @@ def cli_run_url(url, service):
     config.run_url(url, service)
 
 
+@cli.command("init")
+@click.option("-t", "--token")
```

Comment:
Short option is not needed here, it might be even confusing in this case (i.e. `-t` as in `target`)